SUPPORTED-HARDWARE.md

```diff
@@ -1,12 +1,28 @@
 
-This document captures the hardware Bottlerocket for bare metal is tested on and known to work with.
-It also contains hardware that contributors have reported to be functional.
+With the introduction of the metal variant of Bottlerocket we first aimed at providing compatibility with standard x86 server hardware.
+For the initial offering we aimed at a wider spread of 10G+ ethernet NIC drivers in their base configuration (no model specific FPGA offloading support and similar) as well as common RAID controllers.
+We have cut down on many things beyond that, that you would find in a general purpose Linux distribution. The aim is to keep Bottlerocket images as lean as possible, while trying to maintain a good out-of-the-box coverage.
 
-Bottlerocket's kernel configuration isn't as extensive as some other distros, so it's possible that it is missing drivers for your specific hardware.
-If so, please [submit an issue](https://github.com/bottlerocket-os/bottlerocket/issues/new?assignees=&labels=&template=feature.md) and we'll work on integrating the proper kernel modules for your hardware!
+As a result Bottlerocket's kernel configuration isn't as extensive as some other Linux distributions, so it's possible that it is missing drivers for your specific hardware.
+If so, please [submit an issue](https://github.com/bottlerocket-os/bottlerocket/issues/new?assignees=&labels=&template=metal_driver.md) and we'll work on integrating the proper kernel modules for your hardware!
 
-## Confirmed support
+## Limitations of hardware support we can add
+
+* We currently do not plan to add out-of-tree drivers to Bottlerocket images. Maintaining additional driver packages for a probably narrow use-case in the metal variant conflicts with Bottlerocket's core tenets of keeping the it small and simple.
+* We currently do not plan to add additional CPU architectures.
+* We currently do not plan to add drivers for embedded devices in the core images.
+
+However, we are happy to discuss additions of drivers that are part of the Linux source tree. We are also currently working on making it easier to create custom Bottlerocket variants through out-of-tree support that would help to roll a custom variant of Bottlerocket tailored towards your specific use case. That effort can be tracked in [issue #2669](https://github.com/bottlerocket-os/bottlerocket/issues/2669).
+
+## Testing
+
+With the huge variety of hardware we will never be able to do extensive testing on all possible hardware configurations Bottlerocket is compatible with (compatible means the metal variant includes all necessary drivers).
+We are doing some base level functional testing on a limited set of server configurations we have available.
+
+We are interested to learn of your success stories of running Bottlerocket on other hardware platforms. Feel free to report a working configuration below by opening a PR with the info.
+
+### HW configurations confirmed to work by Bottlerocket team at AWS
```

Comment:
- Characters are cheap :) Spell out "Hardware"
- I'm not sure I would organize it this way. Maybe if there is a single table with all the configurations and a column with the confirming organization? (AWS being one). That way if, say, NASA confirms that it works on some hardware they can choose to trust it.

Reply:
That makes a lot of sense. I took the sectioning from what we had before here rephrasing a little, but was not really happy with that separation myself.